language-support/ts/codegen/tests/ts/build-and-lint-test/src/__tests__/test.ts

```diff
@@ -461,24 +461,24 @@ test("multi-{key,query} stream", async () => {
 });
 
 test('stream close behaviour', async () => {
+  const waitTime = 100;
   const url = 'ws' + httpBaseUrl().slice(4) + 'v1/stream/query';
   const events: string[] = [];
   events.push('before creation');
   const ws = new WebSocket(url, ['jwt.token.' + ALICE_TOKEN, 'daml.ws.auth']);
   events.push('after creation');
-  await sleep(10);
```

Comment:
In case anyone wonders, with this we can end up registering the `open` event after it has already fired in which case the handler will never get called. Without this, we never yield control so this is guaranteed to not be racy.